operator/metadata/updater_test.go

```diff
@@ -0,0 +1,138 @@
+package metadata
```

Comment:
Unit tests for `RetrieveInitialMetadata` and `Stream` are still to be implemented

Reply:
implemented